index.html

```diff
@@ -194,9 +219,10 @@ <h4>
                 </li>
               </ol>
             </li>
-            <li>If a file type is being blocked due to security considerations,
-            return <a>a promise rejected with</a> with a {{"NotAllowedError"}}
-            {{DOMException}}.
+            <li>If |data|'s {{ShareData/files}} member is present and
+            {{Navigator/canShareFiles()}} returns false when called with the
+            files' extensions and content types, return <a>a promise rejected
+            with</a> with a {{"NotAllowedError"}} {{DOMException}}.
```

Comment:
I think this should be a {{TypeError}} instead, as the type is not allowed to be shared.